packages/server/src/sync/schema.ts

```diff
@@ -2,8 +2,7 @@ import { Channel } from '@botpress/messaging-channels'
 import Joi from 'joi'
 
 const SyncWebhookSchema = Joi.object({
-  url: Joi.string().uri().required(),
-  token: Joi.string()
```

Comment:
As discussed, we should not let the user send webhook tokens when syncing its configs.